.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.idea/
```

Comment:
Sorry, this line should be deleted I added it so It wouldn't add the WebStorm idea config to git